### ISSUE TITLE
commentShortcuts feature not working

### ISSUE THREAD

**Issue by SmartManoj** [soscripted/sox#420]
The "Use Ctrl+I,B,K (to italicise, bolden and add code backticks) in comments" feature has an error:

```
Uncaught TypeError: $(...).surroundSelectedText is not a function
    at HTMLTextAreaElement.eval (userscript.html?id=cf7158da-e94f-410f-9033-e1aedd3cbc41:1556)
    at HTMLDivElement.dispatch (userscript.html?id=cf7158da-e94f-410f-9033-e1aedd3cbc41:7)
    at HTMLDivElement.y.handle (userscript.html?id=cf7158da-e94f-410f-9033-e1aedd3cbc41:7)
```
---

### Features Enabled 

    ["Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-dragBounty","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-metaNewQuestionAlert","Appearance-scrollToTop","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-openImagesAsModals","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideChatSidebar","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Voting-betterCSS","Voting-stickyVoteButtons","Voting-disableVoteButtons","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksPrivacy","Extras-sortByBountyAmount","Extras-warnNotLoggedIn","Extras-hideCertainQuestions","Extras-showMetaReviewCount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue"]

**Comment by SmartManoj** [comment]
Add icon to each feature to quickly report.
How to find the latest dev version?

**Comment by shu8** [comment]
Thanks for the bug report, fixed in [dev 2.5.22](https://github.com/soscripted/sox/raw/dev/sox.user.js)!

>How to find the latest dev version?
The `@updateURL` in the `sox.user.js` file links to the latest dev version, or just install it from the dev branch in this repo. The link I linked above is a direct link for the latest dev version at any time :)
